src/Utilities/Compiler/Lightup/IsFileLocalWrapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+#if HAS_IOPERATION
+
+namespace Analyzer.Utilities.Lightup
+{
+    using System;
+    using System.Diagnostics.CodeAnalysis;
+    using Microsoft.CodeAnalysis;
+
+    [SuppressMessage("Performance", "CA1815:Override equals and operator equals on value types", Justification = "Not a comparable instance.")]
+    internal readonly struct IsFileLocalWrapper
```

Comment:
💡 This should be a static class with an extension method named `IsFileLocal`:

```cs
public static bool IsFileLocal(this INamedTypeSymbol symbol)
  => s_isFileLocal(symbol);
```

Reply:
I added it to the existing `INamedTypeSymbolExtensions` class, I can add create a new one in the Lightup namespace if that's more desirable

Reply:
This was a fine approach 👍